DynamicBackground/src/home.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
+import './home.css'
+export const Home = () => {
+    const [rows, setRows] = useState(3);
+    const [columns, setColumns] = useState(3);
+    const [colors, setColors] = useState('#FF5733, #33FF57, #5733FF', '#F78F26');

```

Comment:
```suggestion
    const [colors, setColors] = useState('#FF5733, #33FF57, #5733FF', '#F78F26');

```
something here will cause error, find it. do not push code with error ever in PR.